Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
I think the most recent is ubi9. Maybe we can consider start using it. Or do we have some constrain for it to be 8?

Reply:
I can go with ubi9